docs/apis/faucet.md

```diff
@@ -0,0 +1,97 @@
+# Faucet API
+
+Faucet endpoint allows requesting funds from the Faucet.
+
+The API provides three functions to interact with this primitive layer:
+* [/faucet](#faucetrequest)
+
+
+Client lib APIs:
+* [SendFaucetRequest()](#faucetrequest)
+
+
+
+## `/faucet`
+
+POST request asking for funds from the faucet to be transferred to address in the request.
+
+### Method 
+
+`POST`
+
+### Parameters
+
+| **Parameter**            | `address`      |
+|--------------------------|----------------|
+| **Required or Optional** | required       |
+| **Description**          | address to pledge funds to  |
+| **Type**                 | string      |
+
+
+| **Parameter**            | `accessManaPledgeID`      |
+|--------------------------|----------------|
+| **Required or Optional** | optional       |
+| **Description**          | node ID to pledge access mana to  |
+| **Type**                 | string      |
+
+| **Parameter**            | `consensusManaPledgeID`      |
+|--------------------------|----------------|
+| **Required or Optional** | optional       |
+| **Description**          | node ID to pledge consensus mana to  |
+| **Type**                 | string      |
+
+| **Parameter**            | `powTarget`      |
+|--------------------------|----------------|
+| **Required or Optional** | required       |
+| **Description**          | target Proof of Work difficulty |
+| **Type**                 | int      |
+
+
+#### Body
```

Comment:
`powTarget` is missing, same goes to the cURL example.

Reply:
powTarget is only passed in client lib, because pow is calculated on the client side in this case. I clarified it.